test/com/facebook/buck/rules/macros/testdata/query-paths/BUCK.fixture

```diff
@@ -20,5 +20,5 @@ genrule(
 genrule(
     name = "files",
     out = "files.txt",
-    cmd = "echo $(query_paths 'inputs(deps(:example))') >$OUT",
+    cmd = 'PATHS=($(query_paths "inputs(deps(:example))")); for p in "${PATHS[@]}"; do echo "$p"; done | sort > $OUT',
```

Comment:
This is not compatible with Windows

Reply:
Updated to read from the filelist instead